### PR TITLE
[DOJOT-8] [DOJOT-9] [DOJOT-11] Object storage and retrieval

### PR DESCRIPTION
**Pull request by rodrigoaasm**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

[DOJOT-8]

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
[DOJOT-8]: https://dojot.atlassian.net/browse/DOJOT-8
[DOJOT-9]: https://dojot.atlassian.net/browse/DOJOT-9
[DOJOT-11]: https://dojot.atlassian.net/browse/DOJOT-11

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: storage/file/minio/file-mgmt/src/services/tenant-service.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    this.minioRepository = minioRepository;
+    this.logger = logger;
+  }
+
+  /**
+   * Creates a bucket for the new tenant.
+   *
+   * @param {*} bucketName Bucket name
+   */
+  create = async (bucketName) => {
```
Comment (rascaraficci) [inline storage/file/minio/file-mgmt/src/services/tenant-service.js:17]: Não deveríamos remover o bucket em eventos de remoção do tenant?

**Review by rascaraficci — COMMENTED** [review]
Great job! I left some comments!

File: storage/file/minio/file-mgmt/tests/integration/config/default.conf [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+subscribe.topics.regex.tenants:string=dojot.tenancy
+
+#MinIO
+minio.host:string=localhost
+minio.port:integer=9010
+minio.ssl:boolean=false
+minio.accessKey:string=bac03160842a805864ce88b0ba74667c
+minio.secretKey:string=a2ffda15c2168e3954159e19260c4994024916ea
+minio.bucket.suffix:string=cpqd.dojot.
+minio.upload.size.limit:integer=2048
```
[the remark this answers is not in the document]
Comment (rodrigoaasm) [reply storage/file/minio/file-mgmt/tests/integration/config/default.conf:22]: Essa é a configuração dos testes de integração, onde o limite de upload foi reduzido para que não houvesse a necessidade de inserir um arquivo maior que 25mb no repositório.

File: storage/file/minio/file-mgmt/docs/v1.yml [hunk 3]
```diff
@@ -0,0 +1,488 @@
...
+              schema:
+                $ref: "#/components/schemas/apiErrorUnknown"
+
+  "/api/v1/files?alt=url":
+    get:
+      tags:
+        - files
+      security:
+        - bearerAuth: []
+      summary: "Retrieves a URL to extern download"
```
[the remark this answers is not in the document]
Comment (rodrigoaasm) [reply storage/file/minio/file-mgmt/docs/v1.yml:153]: U com pronuncia de "you".

**Review by rascaraficci — APPROVED** [review]
It looks good to me!

**Review by luissajunior — APPROVED** [review]

**Pull request merged**